pkg/kinflate/resource/resourcecollection.go

```diff
@@ -0,0 +1,41 @@
+package resource
+
+import (
+	"fmt"
+
+	manifest "k8s.io/kubectl/pkg/apis/manifest/v1alpha1"
+	"k8s.io/kubectl/pkg/kinflate/types"
+	"k8s.io/kubectl/pkg/loader"
+)
+
+type ResourceCollection types.KObject
+
+// NewRCFromManifest returns a ResourceCollection given a manifest object
+func NewRCFromManifest(l loader.Loader, m *manifest.Manifest) (ResourceCollection, error) {
```

Comment:
Not happy with functions in resource package having knowledge of manifest.

Reply:
Yeah, I'm getting more convinced that we should remove the entire `manifest` package from the resource package.